virtcontainers/kata_agent.go

```diff
@@ -1550,6 +1556,61 @@ func (k *kataAgent) handleEphemeralStorage(mounts []specs.Mount) []*grpc.Storage
 	return epheStorages
 }
 
+// handleHugePages handles hugepages storage by
+// creating a Storage from corresponding source of the mount point
+func (k *kataAgent) handleHugepages(mounts []specs.Mount, hugepageLimits []specs.LinuxHugepageLimit) []*grpc.Storage {
+	//Map to hold the total memory of each type of hugepages
+	optionsMap := make(map[string]string)
+	var HugePageSizeUnitList = []string{"B", "K", "M", "G", "T", "P"}
+
+	for _, hp := range hugepageLimits {
+		if hp.Limit != 0 {
+			k.Logger().WithField("Pagesize", hp.Pagesize).Info("hugepage request")
+			k.Logger().WithField("Limit", hp.Limit).Info("hugepage request")
```

Comment:
Please don't split logical log entries across multiple log lines, just do something like:

```go
k.Logger().WithFields(logrus.Fields{
    "Pagesize": hp.Pagesize,
    "Limit" : hp.Limit.
}).Info("hugepage request")
```

Reply:
Sure.. I'll change it